docs/PLATFORMS.md

```diff
@@ -0,0 +1,64 @@
+openFrameworks currently supports the following platforms:
+
+# Android #
+
+versions: ?
+
+architectures:
+  * armeabi-v7a
+  * i386
```

Comment:
or better perhaps change i386 to x86 it's wat we use in most cases in the folders